demos/prefsFramework/js/SelfVoicingEnactor.js

```diff
@@ -34,6 +34,9 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         model: {
             value: false
```

Comment:
Now that we have moved exclusively to the new relay system, we can eliminate all of this wrapping with the spurious path "value" everywhere - all of these cooperating components can save on configuration and runtime garbage by simply declaring model: false etc.